indigo_api/models/tasks.py

```diff
@@ -86,6 +86,12 @@ class Task(models.Model):
         ('review-work-expression', __('Sign-off')),
     ]
 
+    MAIN_CODES = [
```

Comment:
To avoid repeating yourself, declare these before CODES and then do something like:

```
CODES = MAIN_CODES + [...]
```